### PR TITLE
Migrate to jsbundling

### PR DESCRIPTION
**Pull request by kimadactyl**

### CODE REVIEW

**Comment by alexandria-gfsc** [comment]
LGTM :)

File: app/views/layouts/admin/application.html.erb [hunk 1]
```diff
@@ -4,11 +4,11 @@
     <title><%= "#{yield(:title)} | " if content_for?(:title) %>PlaceCal Admin</title>
     <%= csrf_meta_tags %>
 
-    <%= stylesheet_link_tag 'admin', 'https://fonts.googleapis.com/css?family=Raleway:500,700,800|Trocchi', media: 'all', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag 'admin', 'https://fonts.googleapis.com/css?family=Raleway:500,700,800|Trocchi', media: 'all', 'data-turbo-track': 'reload' %>
 
-    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_pack_tag 'admin', 'data-turbolinks-track': 'reload' %>
-    <%= stylesheet_pack_tag 'admin', 'data-turbolinks-track': 'reload' %>
+    <%# javascript_include_tag 'application', 'data-turbo-track': 'reload', defer: true %>
```
Comment (alexandria-gfsc) [inline app/views/layouts/admin/application.html.erb:9]: any rationale behind this being commented out btw?

File: bin/dev [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+if ! command -v foreman &> /dev/null
+then
+  echo "Installing foreman..."
+  gem install foreman
+fi
```
Comment (alexandria-gfsc) [inline bin/dev:7]: lol this didn't actually work for me and i had to do it manually

Comment (alexandria-gfsc) [reply bin/dev:7]: dunno why!

File: config/cable.yml [hunk 3]
```diff
@@ -1,5 +1,6 @@
 development:
-  adapter: async
+  adapter: redis
+  url: redis://localhost:6379/1
```
Comment (alexandria-gfsc) [inline config/cable.yml:3]: This seems like it will require redis in the server environment?

**Review by alexandria-gfsc — APPROVED** [review]
LGTM and we'll see if the deploy messes up, probably best to apply server changes based on what the error log says :+1:

File: app/views/layouts/admin/application.html.erb (hunk 1 above)
Comment (kimadactyl) [reply app/views/layouts/admin/application.html.erb:9]: i changed them to exist independantly of each other, could remove this line now tbh

File: bin/dev (hunk 2 above)
Comment (kimadactyl) [reply bin/dev:7]: weird!!

File: config/cable.yml (hunk 3 above)
Comment (kimadactyl) [reply config/cable.yml:3]: i think only if we use something that uses redis (i.e. actioncable), otherwise im assuming this gets ignored

**Pull request merged**